calibration/CTDMOG/CGINS-CTDMOG-13579__20170401.csv

```diff
@@ -15,11 +15,11 @@ serial,name,value,notes
 37-13579,CC_pa2,-3.780214e-011,
 37-13579,CC_ptca0,5.250413e+005,
 37-13579,CC_ptca1,-3.730197e+000,
-37-13579,CC_ptca2,9.546746e-002,
+37-13579,CC_ptca2,9.546745e-02,
 37-13579,CC_ptcb0,2.532288e+001,
 37-13579,CC_ptcb1,-2.250000e-004,
 37-13579,CC_ptcb2,0.00E+00,
 37-13579,CC_ptempa0,1.838072e+002,
 37-13579,CC_ptempa1,-5.884309e-002,
 37-13579,CC_ptempa2,-2.246901e-006,
-37-13579,CC_wbotc,1.2350e-006,
+37-13579,CC_wbotc,1.234992e-06,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT